graylog2-web-interface/src/components/widgets/Widget.jsx

```diff
@@ -1,21 +1,22 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import ReactDOM from 'react-dom';
 import $ from 'jquery';
 import { PluginStore } from 'graylog-web-plugin/plugin';
-
 import { WidgetConfigModal, WidgetEditConfigModal, WidgetFooter, WidgetHeader, WidgetVisualizationNotFound } from 'components/widgets';
-
 import StoreProvider from 'injection/StoreProvider';
-const WidgetsStore = StoreProvider.getStore('Widgets');
-
 import ActionsProvider from 'injection/ActionsProvider';
-const WidgetsActions = ActionsProvider.getActions('Widgets');
-
 import Routes from 'routing/Routes';
 import EventHandlersThrottler from 'util/EventHandlersThrottler';
+import Reflux from 'reflux';
```

Comment:
Module imports are placed at the top of the imports. Usually it's react, followed by reflux (if required), then others.